cmd/trade.go

```diff
@@ -64,6 +62,13 @@ func init() {
 		}
 		log.Println(startupMessage)
 
+		if *logPrefix != "" {
+			t := time.Now().String()
```

Comment:
the time string produces a weird long filename with spaces. example:
`test_2018-10-24 15:43:04.414657477 -0700 PDT m=+0.003531672`

this should produce a timestamp in your local timezone formatted like so (year, month, day, T, hour, min, second, tz):
`test_20181024T154304PDT`

see the [Time.Format](https://golang.org/pkg/time/#Time.Format) function in the _time_ package for reference

Reply:
can you also give the filename a `.log` extension

Reply:
I implemented this without making the issuers explicitly optional to avoid bulky conditional code, same approach as the later `"Trading %s:%s for %s:%s\n"` message. This results in a double underscore when XLM is an asset , i.e `XLM__BTC` but I don't think that's a problem.